tests/runtime/test_grpc_server.py

```diff
@@ -374,6 +378,8 @@ def test_train_fake_module_does_not_change_another_instance_model_of_block(
         sample_input_sampleinputtype=SampleInputType(name="Gabe").to_proto(),
         batch_size=100,
         training_data=training_data,
+        training_parameters_json_dict=training_parameters_json_dict,
+        training_parameters=training_parameters,
```

Comment:
Kinda related to the other comment about moving these to the primitive test module, this particular test is already pretty dang beefy and according to the name is testing some model management logic. I think it would be better not to add extra potential failure cases to this test